edgelet/build/windows/package.ps1

```diff
@@ -69,11 +101,15 @@ Function New-Package([string] $Name, [string] $Version)
         Throw "Failed to expand cab"
     }
     Remove-Item -Path $EdgeCab
+    if($Arm -and (-not [string]::IsNullOrEmpty($oldPath)))
+    {
+        $env:Path = $oldPath
+    }
 }
 
 if ($CreateTemplate) {
-    $docker_cli_uri = 'https://mby.blob.core.windows.net/mby-win-amd64/docker-3.0.5.exe'
-    $docker_cli_license_uri = 'https://mby.blob.core.windows.net/mby/LICENSE-cli'
+    $docker_cli_uri = if($Arm) { 'https://edgebuild.blob.core.windows.net/iotedge-win-arm32v7-tools/docker.exe'} else { 'https://mby.blob.core.windows.net/mby-win-amd64/docker-3.0.5.exe' }
+    $docker_cli_license_uri = if ($Arm) { 'https://edgebuild.blob.core.windows.net/iotedge-win-arm32v7-tools/dockerd.exe' } else { 'https://mby.blob.core.windows.net/mby/LICENSE-cli' }

```

Comment:
`dockerd.exe` URL is for `docker_engine_uri`, not `docker_cli_license_uri`. Bad merge?